components/cam/src/dynamics/se/stepon.F90

```diff
@@ -290,7 +296,8 @@ subroutine stepon_run2(phys_state, phys_tend, dyn_in, dyn_out )
          call edgeVpack_nlyr(edge_g,dyn_in%elem(ie)%desc,dyn_in%elem(ie)%derived%FQ(:,:,:,:),nlev*pcnst,kptr,nlev_tot)
 
       end do ! ie
-   endif ! .not. single_column
+
+10 continue
```

Comment:
Do we really want to use goto statements when we can still use the if statement above?  Same comment for the next change below.

Reply:
Agreed, I do not think we want goto statements here.